back-end/server.py

```diff
@@ -82,10 +83,10 @@ def run(self, request):
                 username=self.username
             )
             return ("Success: workflow pushed to {}".format(
-                            request.json['repo']), 200)
+                            request.json['repo']), HTTPStatus(200))
 
         except errors.ServiceError as e:
-            return (e, 400)
+            return (e, HTTPStatus(417))
```

Comment:
We are not dealing with the `Expect request-header field`, so just return `500`.

Reply:
Changed and pushed.